docs/source/models/cause.rst

```diff
@@ -298,9 +285,24 @@ scenarios:
   location
 - A simulant enters the simulation by *aging* into the simulation
 
-Prevalence is **not** used to initialize cause model states when a simulant 
-is *born* into a simulation. See the below section on birth prevalence for 
-how cause model states are initialized in this scenario.
+	Prevalence is **not** used to initialize cause model states when a 
+	simulant is *born* into a simulation. See the below section on birth 
+	prevalence for how cause model states are initialized in this scenario.
+
+While GBD results are estimates of *point* prevalence at the year midpoint, 
+cause model initialization occurs continuously throughtout a full simulation 
+year. The following are two possible assumptions on how the GBD point 
+prevalence estimate might apply across the full simulation year that it 
+represents.
+
+- Point prevalence estimate is constant across the full year. This is likely 
+  appropriate in cases where prevalence is relatively constant over time. 
+
+- Prevalence estimates across several years follow a smooth line between 
```

Comment:
Eh, maybe.  There are both age and time axes to consider here when we're talking about what we should fit a line too.  I'll draw a picture at coworking.

Reply:
Updated in last commit!